recipes-core/udev/files/toradex-net-rename.sh

```diff
@@ -3,57 +3,68 @@
 # Assign name "ethernetN" to net devices with Toradex OUI.
 # Supports Toradex boards with single/dual ethernet interfaces.
 
-if [ $# -ne 1 ]; then
-    echo "Usage: $0 KERNEL" >&2
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 OUI KERNEL" >&2
     exit 1
 fi
 
-dev="${1}"
+mac_oui="${1}"
```

Comment:
@EdTheBearded Any particular reason for not passing the OUI here directly (as the usage suggests)?

Reply:
Done!